### ISSUE TITLE
[5.0.1] New `withXForwardedFor` autocomplete builder method results in incorrect HTTP header

### ISSUE THREAD

**Issue by shengslogar** [smartystreets/smartystreets-php-sdk#56]
https://github.com/smartystreets/smartystreets-php-sdk/commit/0dc5c313641a42998103afb74d3b190d82773267#diff-2258d1b1c88120c093643b76c98deca30e816a85c7e3f9788be6b8013a6c85edR85-R88 introduced the `withXForwardedFor` method to allow passing an IP address to Smarty's autocomplete API for geographically-adjacent suggestions ([related comment/issue](https://github.com/smartystreets/smartystreets-php-sdk/issues/48#issuecomment-2110542255)).

In testing, this has no effect on API results.

https://github.com/smartystreets/smartystreets-php-sdk/blob/41f35cbb91d3a4a9d780cd76315aaeaba165c901/src/NativeSender.php#L85-L88

Changing the HTTP header results in an API response as expected. (Although [the docs specify `X-Forwarded-At`](https://www.smarty.com/docs/cloud/us-autocomplete-pro-api#prefer-geolocation), casing does not appear to be important.)
```diff
if ($this->ip != null) {
---   curl_setopt($ch, CURLOPT_HTTPHEADER, array("X_FORWARDED_FOR: $this->ip"));
+++   curl_setopt($ch, CURLOPT_HTTPHEADER, array("X-FORWARDED-FOR: $this->ip"));
      $smartyRequest->setHeader('X_FORWARDED_FOR', $this->ip);
}
```

**Comment by RyanLCox1** [comment]
Thank you! This has been fixed with the release of 5.0.2

**Comment by shengslogar** [comment]
Thanks for the quick patch!
